### ISSUE TITLE
Adding a CidrAddressField to a model causes a 'blank' admin page

### ISSUE THREAD

**Issue by darkpixel** [jimfunk/django-postgresql-netfields#27]
I have a very simple model:
```
class Network(BaseModel):
    block = CidrAddressField()
    office = models.ForeignKey(Office, null=True, blank=True)
    description = models.CharField(max_length=255, null=True, blank=True)
```

...and it is registered in admin.py:

```
@admin.register(Network)
class NetworkAdmin(BaseAdmin):
    list_display = ['block', 'office', 'description']
    fields = ['office', 'block', 'description']
```

When I attempt to view it in the admin section, I see the following:
![screenshot from 2015-01-14 13 02 50](https://cloud.githubusercontent.com/assets/158380/5747631/5875cac4-9bee-11e4-8255-346aaea7eb0b.png)

When I change the NetworkAdmin object to remove the 'block' field like so:
```
@admin.register(Network)
class NetworkAdmin(BaseAdmin):
    list_display = ['block', 'office', 'description']
    fields = ['office', 'description']
```
I see:
![screenshot from 2015-01-14 13 04 00](https://cloud.githubusercontent.com/assets/158380/5747650/7e6788b2-9bee-11e4-8409-5819e9d7b382.png)

The 'list view' display in admin works fine displaying the 'block' field, it just doesn't show up when I want to edit.

I've tried overriding the form generated by admin to specifically use the AdminTextInputWidget for the 'block' field, and it still shows up as a blank page.

Am I missing something?

**Comment by jimfunk** [comment]
I can't reproduce this with the model inheriting from models.Model and the admin inheriting from admin.ModelAdmin. Perhaps this is caused by the base models you are using?

**Comment by darkpixel** [comment]
Sorry, I should have included the base models as well.
The BaseModel is just a simple model to track created/modified times for objects.
BaseAdmin sets a few features I consider 'nice' in django-admin.

#models.py
```
class BaseModel(models.Model):                                                                                              
                                                                                                                            
    """ BaseModel provides information on creation and modification times for all child objects. """                        
                                                                                                                            
    created = models.DateTimeField(editable=False, blank=True, null=True)                                                   
    modified = models.DateTimeField(editable=False, blank=True, null=True)                                                  
                                                                                                                            
    def save(self, *args, **kwargs):                                                                                        
        """ Save an object and note the created or modified time. """                                                       
        if not self.pk:                                                                                                     
            self.created = timezone.now()                                                                                   
        self.modified = timezone.now()                                                                                      
        super(BaseModel, self).save(*args, **kwargs)                                                                        
                                                                                                                            
    class Meta:                                                                                                             
        abstract = True                                                                                                     
```

#admin.py
```
class BaseAdmin(admin.ModelAdmin):

    """ Base module inherited by all other admin modules to save time setting various defaults. """

    actions_on_top = True
    actions_on_bottom = True
    save_as = True
    save_on_top = True

```

**Comment by jimfunk** [comment]
I added this stuff and it still works properly for me. Are you using the current code? Are there any global admin settings perhaps interfering?

**Comment by darkpixel** [comment]
I am not running the latest version.  Doing a 'pip install --upgrade django-netfields' causes Django to be upgraded to 1.8.2 which is also currently incompatible with django-celery-haystack.  I'm stuck on Django 1.7.6 if I want search to work.  Or at least that was the cause about 3 months ago.  I'll check again tomorrow and see if I can update my toolchain.

**Comment by jimfunk** [comment]
You  can install without upgrading Django by installing django-netfields==0.3.1

**Comment by darkpixel** [comment]
This is fixed in 0.3.1

Thanks.
